x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -211,7 +211,7 @@ export const allowedExperimentalValues = Object.freeze({
   /**
    * Enables visualization: session viewer and analyzer in expandable flyout
    */
-  visualizationInFlyoutEnabled: false,
+  visualizationInFlyoutEnabled: true,
```

Comment:
@christineweng was this change meant to be merged to `main`?

Reply:
no, this is to setup a cloud instance for a customer meeting Paul has today. Will revert before merging